versions/latest/modules/en/pages/installation-and-upgrade/infrastructure-setup/configure-oidc-provider.adoc

```diff
@@ -0,0 +1,141 @@
+= Configure Rancher as an OIDC provider
+:revdate: 2025-08-27
+:page-revdate: {revdate}
+
+Rancher can function as a standard OpenID Connect (OIDC) provider, allowing external applications to use Rancher for authentication.
+This can be used for enabling single sign-on (SSO) across Rancher Prime components. For example, see the https://documentation.suse.com/cloudnative/suse-observability/next/en/setup/security/authentication/oidc.html[documentation] for configuring the OIDC provider for SUSE Observability.
```

Comment:
```suggestion
This can be used for enabling single sign-on (SSO) across Rancher Prime components. For example, see the https://documentation.suse.com/cloudnative/suse-observability/latest/en/setup/security/authentication/oidc.html[documentation] for configuring the OIDC provider for SUSE Observability.
```

We can should skip the redirect.